dashboard/test/ui/features/teacher_tools/submittable_eyes.feature

```diff
@@ -11,6 +11,7 @@ Scenario: Submittable level
   And I click selector ".submitButton"
 
   Then I am on "http://studio.code.org/s/allthethings/lessons/9/levels/3?noautoplay=true"
+  Then I reload the page
```

Comment:
I can see why this would make sure they are on the right page, but perhaps isn't authentic to the what the user would actually do.  

I just wanted to make sure this was an intentional choice on your side to have this line.

Reply:
Such a fair callout- I think this is actually a really non-authentic user flow. What we're trying to show with this test is that if the student navigates back to this page, they have the option to unsubmit. The steps it would take to do so would actually be submit->press okay (get redirected) -> click the back button OR press the prior bubble? I think this shortcut is a faster way to get back to the page, and the reload might allow a data refresh the way truly navigating away and back would've.

Reply:
Got it - that makes sense!

Reply:
We had a conversation about this offline- worth a try, so resolving this and merging